src/views/product-landing/components/get-started-card/get-started-card.module.css

```diff
@@ -0,0 +1,9 @@
+.placeholder {
+  border: 1px solid magenta;
+  font-size: 12px;
+  white-space: pre-wrap;
+}
+
+.heading {
+  composes: g-offset-scroll-margin-top from global;
```

Comment:
Note that stubbing out placeholder components is also helpful for getting the table-of-contents to function correctly off the bat. Re-using `g-offset-scroll-margin-top` here, which was introduced in #377 .